.eslintrc.json

```diff
@@ -23,16 +23,7 @@
     }],
     "space-before-function-paren": ["error", "never"],
     "quotes": [1, "single", "avoid-escape"],
-    "comma-dangle": ["error", "always-multiline"],
-    "flowtype/define-flow-type": 1,
-    "flowtype/require-parameter-type": [1, { "excludeArrowFunctions": true }],
-    "flowtype/require-return-type": [ 0,"always", { "annotateUndefined": "never" } ],
-    "flowtype/space-after-type-colon": [ 1, "always" ],
-    "flowtype/space-before-type-colon": [ 1, "never" ],
-    "flowtype/type-id-match": [ 0, "^([A-Z][a-z0-9]+)+Type$" ],
-    "flowtype/use-flow-type": 1,
-    "flowtype/valid-syntax": 1,
-    "react/jsx-uses-react": "error"
+    "comma-dangle": ["error", "always-multiline"]
```

Comment:
the "extends" plugin thing is new, which is why we don't use in emission - ^ these are all basically the recommended anyway